index.js

```diff
@@ -71,20 +71,31 @@ function convertCrontabs() {
         event.schedule.hasOwnProperty("timezone")
       ) {
         const schedule = event.schedule;
-        const match = schedule.rate.match(/^cron\((.*)\)$/);
-        if (!match)
+        const isNewServerlessVersion = Array.isArray(schedule.rate);
```

Comment:
why is that property called `isNewServerlessVersion` - it's quite ambiguous meaning - could we make this more clear?

Reply:
Do you have any suggestions? Would a comment explaining it be enough?
It's being used to switch between the current behaviour and what needs to change for newer Serverless versions.
Is there a better way to encapsulate that?

Reply:
I would name it like `hasRateDefinedAsArray` and add an explanation with a comment above it - what do you think?

Reply:
I've had a stab at it, should I change it again?